utilities/ubi8-google-api-python-client/scripts/upload-file-to-google-drive.py

```diff
@@ -0,0 +1,263 @@
+from __future__ import print_function
+import sys, getopt
```

Comment:
just wondering if there is a better place for this script. can it be used by other things? could it live it a repo of python helper scripts?

Reply:
in theory, but long story short, it was living in another internal repo where we didn't want it any longer. that repo uses this image. this script requires having all the google python libraries available. it seems easiest just to keep it with the container. if someone else somewhere down the road wants to use it outside the container they could do the work to move it. but the juice doesn't seem worth the squeeze to put it in some other repo and then import it during image build.